gtest/spec/test_chat.yaml

```diff
@@ -29,7 +29,7 @@ fixtures:
           value: "0x002c707269766174655f6d7367"
         destination: 2
     expected:
-      - step: 4
+      - step: 6
```

Comment:
why has it changed?

Reply:
This test hadn't taken into account two messages sent from `bot`'s `init` function`. I don't know why this worked before.